utils/appliance/__init__.py

```diff
@@ -142,6 +148,36 @@ def uninstall_ipa_client(self):
         assert return_code != 0
 
 
+class ApplianceLoggerAdapter(logging.LoggerAdapter):
```

Comment:
can we generalize tagged log adaption?
where tag == adding `[...]` in front of messages, adding a spacer if necessary